tests/test_rpqueue.py

```diff
@@ -12,9 +12,11 @@
 rpqueue.log_handler.setLevel(rpqueue.logging.CRITICAL)
 
 queue = b'TEST_QUEUE'
+queue_str = 'TEST_QUEUE'
```

Comment:
Is this variable needed now that you decided to call `.encode()` in the task decorator above?

Reply:
Yes. Decorator except for two types now: bytes and strings. But we encode strings before sending queue name to Redis.